modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/function/ZoomLevelFunction.java

```diff
@@ -60,7 +60,9 @@
                     parameter("scaleDenominator", Number.class),
                     parameter("srid", String.class));
 
-    public static final double EPSG_3857_O_SCALE = 559_082_263.9508929;
+    public static final int ROOT_TILE_PIXELS =
+            Integer.valueOf(System.getProperty("MBSTYLE_ROOT_TILE_PIXELS", "512"));
```

Comment:
Can you add this to the docs somewhere? Config variables that are only in the code (and release announcements) are a pain to track down later on.

Reply:
Done